Extensions/RepeatEveryXSeconds.json

```diff
@@ -0,0 +1,684 @@
+      "functionType": "Condition",
+      "name": "Repeat",
+      "private": false,
+      "sentence": "Repeat timer _PARAM1_ every _PARAM2_ seconds",
+      "events": [
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [
+            {
+              "type": {
+                "inverted": false,
+                "value": "BuiltinCommonInstructions::Once"
+              },
+              "parameters": [],
+              "subInstructions": []
+            }
+          ],
+          "actions": [
+            {
+              "type": {
+                "inverted": false,
+                "value": "ResetTimer"
+              },
+              "parameters": [
+                "player",
+                "GetArgumentAsString(\"timerName\")"
+              ],
+              "subInstructions": []
+            }
+          ],
+          "events": []
+        },
+        {
+          "disabled": false,
+          "folded": false,
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [
+            {
+              "type": {
+                "inverted": false,
+                "value": "Timer"
+              },
+              "parameters": [
+                "player",
+                "GetArgumentAsNumber(\"time\")",
+                "GetArgumentAsString(\"timerName\")"
+              ],
+              "subInstructions": []
+            }
+          ],
+          "actions": [
+            {
+              "type": {
+                "inverted": false,
+                "value": "ResetTimer"
+              },
+              "parameters": [
+                "player",
+                "GetArgumentAsString(\"timerName\")"
+              ],
+              "subInstructions": []
+            },
+            {
+              "type": {
+                "inverted": false,
+                "value": "SetReturnBoolean"
+              },
+              "parameters": [
+                "True"
+              ],
+              "subInstructions": []
+            }
+          ],
+          "events": []
+        }
+      ],
+      "parameters": [
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "The name of the timer to loop",
+          "longDescription": "",
+          "name": "timerName",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "string"
+        },
+        {
+          "codeOnly": false,
+          "defaultValue": "",
+          "description": "The time in seconds between each trigger",
+          "longDescription": "",
+          "name": "time",
+          "optional": false,
+          "supplementaryInformation": "",
+          "type": "expression"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Triggers every X seconds X amount of times.",
+      "fullName": "Repeat every X seconds in X number",
```

Comment:
@arthuro555  I merged this a bit fast, this is not proper english.  Can you fix that?